splunktaucclib/rest_handler/handler.py

```diff
@@ -102,13 +102,39 @@ def check_existing(self, name):
             else:
                 return None
 
+        def basic_name_validation(name):
+            tmp_name = str(name)
+            prohibited_chars = ["*", "\\", "[", "]", "(", ")", "?", ":"]
+            val_err_msg = (
+                (
+                    '"default", ".", "..", string started with "_" and string including any one of '
+                    '["*", "\\", "[", "]", "(", ")", "?", ":"] are reserved value which cannot '
+                    "be used for field Name"
+                ),
+            )
+            if (
+                tmp_name.startswith("_")
+                or tmp_name == "."
+                or tmp_name == ".."
+                or tmp_name == "default"
+            ):
+                raise RestError(400, val_err_msg)
+
+            if any(pc in prohibited_chars for pc in tmp_name):
+                raise RestError(400, val_err_msg)
+
+            if len(tmp_name) >= 1024:
+                raise RestError(400, "Field Name must be less than 1024 characters")
```

Comment:
shorter? 
Or something like "Maximum length of field name is 1024" etc.

Reply:
Sure, we can make it shorter, I just copy paste error msgs from front validation https://github.com/splunk/addonfactory-ucc-generator/blob/6a947e56730699328d2bf499b8cd3e8c017fa39f/ui/src/constants/messageDict.ts#L23